devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy

```diff
@@ -837,6 +855,10 @@ def boseRefreshNowPlaying(delay=0) {
     return boseGET("/now_playing")
 }
 
+def boseSendGetNowPlaying() {
```

Comment:
Yes, it is, it's being setup by the cron job which is polling the device every two minutes, that drives the event that health check is using to see if everything is ok.
If there's events reported in the time configured health check will make a final call to poll() to see if device is around.